nexus/db-model/src/support_bundle.rs

```diff
@@ -0,0 +1,132 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use super::impl_enum_type;
+use crate::schema::support_bundle;
+use crate::typed_uuid::DbTypedUuid;
+
+use chrono::{DateTime, Utc};
+use nexus_types::external_api::shared::SupportBundleInfo as SupportBundleView;
+use nexus_types::external_api::shared::SupportBundleState as SupportBundleStateView;
+use omicron_uuid_kinds::DatasetKind;
+use omicron_uuid_kinds::DatasetUuid;
+use omicron_uuid_kinds::OmicronZoneKind;
+use omicron_uuid_kinds::OmicronZoneUuid;
+use omicron_uuid_kinds::SupportBundleKind;
+use omicron_uuid_kinds::SupportBundleUuid;
+use omicron_uuid_kinds::ZpoolKind;
+use omicron_uuid_kinds::ZpoolUuid;
```

Comment:
We don't enforce this and have plenty of code that does it both ways, but FWIW my preference is for one-import-per-line because it results in much simpler git merges if the imports change in ways that shouldn't conflict (e.g., two branches each adding a new import from the same external crate each rarely conflicts if we're on one-import-per-line, but is basically guaranteed to conflict if the imports are combined since it shifts all the others around).

Reply:
Good point, thanks for the context 👍